src/WeatherOnMarsImage/WeatherOnMarsImage.scss

```diff
@@ -4,6 +4,7 @@
   padding-top: 600px;
   height: 100%;
   width: 100%;
+  z-index: -1; 
```

Comment:
Did not know that negative z-indexes were a thing! (looks like they should work fine though)